casper/src/test/scala/coop/rchain/casper/helper/BlockGenerator.scala

```diff
@@ -95,11 +95,12 @@ trait BlockGenerator {
       blockHash = Blake2b256.hash(header.toByteArray)
       body      = Body().withState(postState).withDeploys(deploys)
       serializedJustifications = justifications.toList.map {
-        case (creator: Validator, latestBlockHash: BlockHash) =>
-          Justification(creator, latestBlockHash)
+        case (cr: Validator, latestBlockHash: BlockHash) =>
```

Comment:
I strongly disagree with making names less obvious. creator is fine.

Reply:
`creator` is used a paramter in name, there was shadowing involved